google/cloud/bigquery/magics/magics.py

```diff
@@ -777,6 +777,12 @@ def _make_bqstorage_client(use_bqstorage_api, credentials, client_options):
         )
         raise customized_error from err
 
+    try:
+        bigquery._helpers._verify_bq_storage_version()
+    except bigquery.LegacyBigQueryStorageError as exc:
+        warnings.warn(str(exc))
+        return None
+
```

Comment:
Would it make sense to replace this and the creation logic on line 794 with a call to `_create_bqstorage_client`?

Reply:
This magics module already contains a bit too much cruft, thus removing some sounds good. :+1: